pootle/apps/pootle_translationproject/models.py

```diff
@@ -277,6 +277,9 @@ def save(self, *args, **kwargs):
         else:
             self.abs_real_path = None
         super(TranslationProject, self).save(*args, **kwargs)
+        if self.directory.tp_id != self.pk:
+            self.directory.tp = self
+            self.directory.save()
```

Comment:
i mean in almost every situation the directory would have been created - but testing brought out a situation where it wouldnt